src/Compilers/Core/Portable/Emit/EditAndContinue/DeltaMetadataWriter.cs

```diff
@@ -784,6 +789,76 @@ private void PopulateEncLogTableParameters()
             }
         }
 
+        private void PopulateEncLogTableCustomAttributes()
```

Comment:
>PopulateEncLogTableCustomAttributes [](http://example.com/codeflow?start=21&length=35)

Perhaps document what our approach to emitting custom attribute delta is here. It's different from other tables so extra documentation would be useful.